ccd-definition/CaseEventToFields/CareSupervision/manageDocuments.json

```diff
@@ -42,67 +42,78 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageDocuments",
-    "CaseFieldID": "manageDocumentsHearingLabel",
+    "CaseFieldID": "manageDocumentsFurtherEvidenceHeading",
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "PageID": "FurtherEvidenceDocuments",
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "N",
-    "PageShowCondition": "manageDocument.type=\"FURTHER_EVIDENCE_DOCUMENTS\"",
-    "FieldShowCondition": "manageDocument.relatedToHearing=\"Yes\"",
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documents/validate-further-evidence/mid-event"
+    "FieldShowCondition": "manageDocument.type=\"FURTHER_EVIDENCE_DOCUMENTS\"",
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documents/validate-supporting-evidence/mid-event"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageDocuments",
-    "CaseFieldID": "furtherEvidenceDocumentsTEMP",
-    "PageFieldDisplayOrder": 2,
-    "DisplayContext": "COMPLEX",
+    "CaseFieldID": "manageDocumentsCorrespondenceHeading",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
     "PageID": "FurtherEvidenceDocuments",
     "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Y"
+    "ShowSummaryChangeOption": "N",
+    "FieldShowCondition": "manageDocument.type=\"CORRESPONDENCE\""
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageDocuments",
-    "CaseFieldID": "correspondenceDocuments",
+    "CaseFieldID": "manageDocumentsC2Heading",
     "PageFieldDisplayOrder": 1,
-    "DisplayContext": "COMPLEX",
-    "PageID": "CorrespondenceDocuments",
-    "PageDisplayOrder": 3,
+    "DisplayContext": "READONLY",
+    "PageID": "FurtherEvidenceDocuments",
+    "PageDisplayOrder": 2,
     "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Y",
-    "PageShowCondition": "manageDocument.type=\"CORRESPONDENCE\"",
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-documents/validate-correspondence-documents/mid-event"
+    "ShowSummaryChangeOption": "N",
+    "FieldShowCondition": "manageDocument.type=\"C2\""
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "manageDocuments",
-    "CaseFieldID": "manageDocumentsSupportingC2Label",
-    "PageFieldDisplayOrder": 1,
+    "CaseFieldID": "manageDocumentsHearingLabel",
+    "PageFieldDisplayOrder": 2,
```

Comment:
I think that's incosistent numbering, is it? It says 2, but the above (C2 and Correspendence type have both 1), is it expected?

Reply:
I'm under the impression @akomisarek that if you have separate numbering, CCD will persevere space for the caseField in the UI regardless of wether it's hidden or not. I would prefer CCD didn't persevere this space for headings that aren't to be shown. Let me test that theory and get back to you.

Reply:
Tested, does not make a difference with the ordering I originally suggested. Updated to reflect your suggestions.